packages/components/src/components/Icon/IconName.ts

```diff
@@ -0,0 +1,4 @@
+import { HeroIconName } from "./HeroIconName";
+import { LucideIconName } from "./LucideIconName";
+
+export type IconName = HeroIconName | LucideIconName;
```

Comment:
There's a few other components that currently use heroicons: Badge, Button, Input. Shouldn't they be updated to have all the same icon options?

Reply:
oh! Good call!